src/test/kotlin/org/jetbrains/research/ml/ast/util/FileTestUtil.kt

```diff
@@ -2,45 +2,35 @@ package org.jetbrains.research.ml.ast.util
 
 import java.io.File
 
+enum class Extension(val value: String) {
+    Py(".py"), Xml(".xml")
+}
+
 object FileTestUtil {
 
     val File.content: String
         get() = this.readText().removeSuffix("\n")
 
     fun getInAndOutFilesMap(
         folder: String,
-        oldExtension: String = ".py",
-        newExtension: String = ".py"
+        inExt: Extension = Extension.Py,
+        outExt: Extension = Extension.Py
     ): Map<File, File> {
-        val inFileRegEx = "in_\\d*$oldExtension".toRegex()
-        val inOutFileRegEx: Regex = "(in|out)_\\d*($oldExtension|$newExtension)".toRegex()
-        val (inFiles, outFiles) = getNestedFiles(folder).toList().filter { inOutFileRegEx.containsMatchIn(it.name) }
-            .partition { inFileRegEx.containsMatchIn(it.name) }
-        if (inFiles.size != outFiles.size) {
-            throw IllegalArgumentException(
-                "Size of the list of input files does not equal size of the list of output files if the folder: $folder"
-            )
-        }
-        return inFiles.associateWith { inFile ->
-            // TODO: can I do it better?
-            val outFileName = inFile.name.replace("in", "out").replace(oldExtension, newExtension)
-            val outFile = File("${inFile.parent}/$outFileName")
-            if (!outFile.exists()) {
-                throw IllegalArgumentException("Out file $outFile does not exist!")
-            }
-            outFile
-        }
-    }
+        fun File.number(): Int = "(?<=(in|out)_)\\d+".toRegex().find(this.name)!!.value.toInt()
+        fun File.isInFile(): Boolean = "in_\\d+.*${inExt.value}".toRegex().containsMatchIn(this.name)
+        fun File.isOutFile(): Boolean = "out_\\d+.*${outExt.value}".toRegex().containsMatchIn(this.name)
```

Comment:
Also maybe should we add new method's arguments in the `getInAndOutFilesMap` function: ` inPrefix: String = "in"` and `outPrefix: String = "out"` and use them in these functions? In this case, we'll be sure to use the same prefixes in these methods.